src/uwtools/api/ungrib.py

```diff
@@ -0,0 +1,53 @@
+"""
+API access to the uwtools ungrib driver.
+"""
+import datetime as dt
+from pathlib import Path
+from typing import Dict, Optional
+
+from uwtools.drivers import support
```

Comment:
```suggestion
from uwtools.drivers import _support
```
We'll want this to be marked private so that API users don't think that `support` is something we intend to be used directly. You'll also need to update the matching reference(s) to `support` below.